### ISSUE TITLE
Add support for rollup@4

### ISSUE THREAD

**Issue by tbouffard** [tivac/rollup-plugin-sizes#174]
Hi and thanks for this plugin

rollup-plugin-sizes@1.0.5 declares `"rollup": "^2 || ^3"` in peerDependencies

Running npm install in a projet using rollup@4 generates the following error
```
 npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: rollup-plugin-sizes@1.0.5
npm ERR! Found: rollup@4.0.2
npm ERR! node_modules/rollup
npm ERR!   dev rollup@"~4.0.2" from the root project
npm ERR!   peerOptional rollup@"^2.68.0||^3.0.0||^4.0.0" from @rollup/plugin-commonjs@25.0.5
npm ERR!   node_modules/@rollup/plugin-commonjs
npm ERR!     dev @rollup/plugin-commonjs@"~25.0.5" from the root project
npm ERR!   5 more (@rollup/plugin-node-resolve, @rollup/plugin-terser, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer rollup@"^2 || ^3" from rollup-plugin-sizes@1.0.5
npm ERR! node_modules/rollup-plugin-sizes
npm ERR!   dev rollup-plugin-sizes@"~1.0.5" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: rollup@3.29.4
npm ERR! node_modules/rollup
npm ERR!   peer rollup@"^2 || ^3" from rollup-plugin-sizes@1.0.5
npm ERR!   node_modules/rollup-plugin-sizes
npm ERR!     dev rollup-plugin-sizes@"~1.0.5" from the root project
```

Possible workaround that could be evaluated (same as in #142): use the `--legacy-peer-deps` option

**Comment by tbouffard** [comment]
I have able to use rollup-plugin-sizes@1.0.5 with rollup@4.1.3 by setting `legacy-peer-deps=true`. See https://github.com/process-analytics/bpmn-visualization-js/pull/2927#pullrequestreview-1679174641

Can other users check that it works on their side to make sure that the plugin works with rollup v4?
Thank you.

**Comment by ricovitch** [comment]
Looks to work for me using legacy-peer-deps option to install.

**Comment by tivac** [comment]
Released as `rollup-plugin-sizes@1.0.6`

**Comment by tbouffard** [comment]
Thanks for merging the PR. I have been able to successfully integrate the new version in my project.
